languages/go/internal/host/host.go

```diff
@@ -329,13 +329,7 @@ func (h Host) ToPolar(v interface{}) (*Value, error) {
 		if err != nil {
 			return nil, err
 		}
-		var repr string
-		value := reflect.ValueOf(v)
-		if value.IsZero() {
-			repr = fmt.Sprintf("%v{}", value.Type())
-		} else {
-			repr = fmt.Sprintf("%v{%v}", value.Type(), v)
-		}
+		repr := fmt.Sprintf("%T%+v", v, v)
```

Comment:
Alternative to `%T%+v` is `%#v`, but I slightly preferred the formatting this way.

### `%T%+v`

```
ZeroFoo{ZeroString: ZeroNumber:0}
Foo{String:hi Number:42}
```

### `%#v`

```
ZeroFoo{ZeroString:\"\" ZeroNumber:0}
Foo{String:\"hi\" Number:42}
```